### ISSUE TITLE
`gofish init` should honor environment variables

### ISSUE THREAD

**Issue by nikaro** [fishworks/gofish#118]
Hello,

```
$ env | grep GOFISH
GOFISH_RIGS=/home/nicolas/.local/share/gofish/rigs
GOFISH_CACHE=/home/nicolas/.cache/gofish
GOFISH_HOME=/home/nicolas/.local/share/gofish
GOFISH_DEFAULT_RIG=/home/nicolas/.local/share/gofish/rigs/github.com/fishworks/fish-food
GOFISH_BARREL=/home/nicolas/.local/share/gofish/barrel

$ gofish init
The following new directories will be created:
/usr/local/Fish
/usr/local/Fish/Barrel
/usr/local/Fish/Rigs
/usr/local/bin
/home/nicolas/.gofish
[sudo] password for root:
```

Also it would be nice if GoFish could have a `GOFISH_BIN` setting to override the default `/usr/local/bin` destination (for example to use `~/.local/bin` and avoid the need of root permissions).

**Comment by nikaro** [comment]
Looks like `gofish home` also does not follow envvars:

```
$ gofish home
/usr/local/Fish
```

**Comment by bacongobbler** [comment]
Good suggestion. I'm marking this as an enhancement if you'd like to take a crack at this.

**Comment by nikaro** [comment]
I'm a very beginner in golang. But i'll take a look at it, this could be an opportunity to get started.

**Comment by bacongobbler** [comment]
implemented in #143. Use `GOFISH_BIN` to change the default bin path.
